examples/cylinder/3d_unsteady_discrete/README.md

```diff
@@ -4,206 +4,27 @@
-
-pde.add_bc("left", bc_left_u, bc_left_v, bc_left_w)
-pde.add_bc("right", bc_right_p)
-pde.add_bc("circle", bc_circle_u, bc_circle_v, bc_circle_w)
-```
-
-Once the equation and the problem domain are prepared, a discretization recipe should be given. 
-This recipe will be used to generate the training data before training starts. 
-
-```
-pde_disc = pde.discretize(
-    time_method="implicit", time_step=1, geo_disc=geo_disc)
-```
-
-### Constructing the neural net
-
-Now the PDE part is almost done, we move on to constructing the neural net.
-It's straightforward to define a fully connected network by creating a `psci.network.FCNet` object.
-Following is how we create an FFN of 5 hidden layers with 20 neurons on each, using hyperbolic
-tangent as the activation function.
-
-```
-net = psci.network.FCNet(
-    num_ins=2,
-    num_outs=3,
-    num_layers=5,
-    hidden_size=20,
-    dtype="float32",
-    activation='tanh')
-```
-
-Next, one of the most important steps is define the loss function. Here we use L2 loss.
-
-```
-loss = psci.loss.L2(p=2)
-```
-
-By design, the `loss` object conveys complete information of the PDE and hence the
-latter is eclipsed in further steps. Now combine the neural net and the loss and we
-create the `psci.algorithm.PINNs` model algorithm.
-
-```
-algo = psci.algorithm.PINNs(net=net, loss=loss)
-```
-
-Next, by plugging in an Adam optimizer, a solver is contructed and you are ready
-to kick off training. In this example, the Adam optimizer is used and is given
-a learning rate of 0.001. 
-
-The `psci.solver.Solver` class bundles the `pde`, which is the training data, the PINNs model, which is called `algo` here,
-and the optimizer, into a solver object that exposes the `solve` interface.
-
+## Baseline
+This guide introduces to how to build a PINN model for simulating the flow around a cylinder in PaddleScience.
+Run the command as follows:
 ```
-opt = psci.optimizer.Adam(learning_rate=0.001, parameters=net.parameters())
-solver = psci.solver.Solver(pde=pde_disc, algo=algo, opt=opt)
+cd baseline
+python3.7 cylinder3d_unsteady.py
 ```
 
-When using `discrete-time method`, the physical information of each time period needs to be solved in turn.
-
-3 parts of dataset need to be added to the solver in each moment. 
-The physical information of the interior point at the current moment is added through `solver.feed_data_interior_cur`. 
-The physical information of the user point at the current moment is added through `solver.feed_data_user_cur`. 
-The physical information of the user point at the next moment is added through `solver.feed_data_user_next`.
-
-After all the data is ready, it can be trained with `solver.solve`. 
-Finally, the physical information of the next moment is solved.
-
+## Optimize
+If you want to try out the advanced automatic differentiation function of static graph, you can run the following command.
+Compared with the baseline version, it can get 15% performance improvement.
```

Comment:
`it can get 15% performance improvement.` 建议修改为
`Due to the new automatic differentiation mechanism, calculation diagram optimization, new actuator execution optimization technology and distributed technology, there are improvements in speed and memory occupation, and can support multi machine and multi card training.`